README.md

```diff
@@ -1,2 +1,137 @@
-# MYCAPSTONE
-My first conference website with  mobile and desktop  view.
+# MYCAPSTONE (Final Capstone Project)
+This is my 1st Capstone project where I used media queries to make website fully responsive in nature. I use javascript event listeners on click event to show speaker section in the home page.
+Used different menubars for mobile and desktop view .
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+- [Install](#install)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [MYCAPSTONE] <a name="about-project"></a>
+.
+**[MYCAPSTONE]** is a simple conference website showin.g details for the upcoming conference in responsive manner having mobile view and desktop view for the home page and about page. 
+
+
+
+## 🛠 Built With <a name="built-with"></a>
+
+- Html
+- CSS
+- Javascript
+
+## 🚀 Project Video <a name="live-video"></a>
+
+- [Project Video Link](https://www.loom.com/share/44ba81f11c0f4a93b1c03d0888b74f9f)
+Here you can see me explaining about my project.
+<!-- LIVE DEMO -->
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://supreeti.github.io/MYCAPSTONE/)
+  Here you can visit my project.
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Setup
+
+Clone this repository to your desired folder:
+
+Example commands:
+
+```sh
+  cd my-folder
+  git@github.com:supreeti/MYCAPSTONE.git
+```
+### Install
+
+Install this project with:
+
+Example command:
+
+```sh
+  cd my-project
+```
```

Comment:
- [optional] Let's work on your README file a bit, kindly provide details on how your beautiful app can be installed and accessed locally. Thanks. 🙏